### PR TITLE
Add F-Droid repo

### PR DESCRIPTION
**Pull request by SailReal**
Screenshot from 2021-04-16 21-54-13

Screenshot from 2021-04-16 21-55-15

Screenshot from 2021-04-16 22-04-36

### CODE REVIEW

**Review by overheadhunter — APPROVED** [review]

File: layouts/android/single.html [hunk 1]
```diff
@@ -4,15 +4,17 @@
...
-    
-    <div class="flex flex-wrap">
-      <section class="w-full py-6 lg:w-1/2 lg:pr-3">
-        {{ partial "android-getkey.html" . }}
-      </section>
 
-      <section class="w-full py-6 lg:w-1/2 lg:pl-3">
-        {{ partial "android-download.html" . }}
-      </section>
+    <div class="grid grid-cols-1 md:grid-cols-2 gap-4 items-start">
```
Comment (tobihagemann) [inline layouts/android/single.html:8]:
```suggestion
    <div class="grid grid-cols-1 md:grid-cols-2 gap-6 items-start">
```

File: layouts/partials/android-download.html [hunk 2]
```diff
@@ -12,7 +12,6 @@
       <h3>{{ i18n "android_download_title" }}</h3>
     </div>
     <div class="px-4 py-8" x-data="{showShaSignature: false}">
-      <img class="inline-block mb-4" src="/img/android/app-icon.png" srcset="/img/android/app-icon.png 1x, /img/android/app-icon@2x.png 2x"/>
```
Comment (tobihagemann) [inline layouts/partials/android-download.html:15]: Check if this asset is used anywhere else. If not, delete both `app-icon.png` and `app-icon@2x.png`.

File: i18n/de.yaml [hunk 3]
```diff
@@ -62,6 +62,13 @@
 - id: android_keyrecovery_contact_us
   translation: "Wenn du keine E-Mail bekommen hast, <a class=\"text-primary no-underline hover:underline\" href=\"mailto:support@cryptomator.org\">kontaktiere uns</a>."
 
+- id: android_fdroid_title
+  translation: "App mittels F-Droid herunterladen"
```
Comment (tobihagemann) [inline i18n/de.yaml:66]:
```suggestion
  translation: "Auf F-Droid herunterladen"
```

File: i18n/de.yaml [hunk 4]
```diff
@@ -62,6 +62,13 @@
...
+- id: android_fdroid_repo
+  translation: "Folgenden <a class=\"text-primary no-underline hover:underline\" href=\"https://static.cryptomator.org/android/fdroid/repo?fingerprint=F7C3EC3B0D588D3CB52983E9EB1A7421C93D4339A286398E71D7B651E8D8ECDD\">Link</a> öffnen oder den QR-Code scannen um unser Repository hinzuzufügen."
```
Comment (tobihagemann) [inline i18n/de.yaml:68]:
```suggestion
  translation: "Folgenden <a class=\"text-primary no-underline hover:underline\" href=\"https://static.cryptomator.org/android/fdroid/repo?fingerprint=F7C3EC3B0D588D3CB52983E9EB1A7421C93D4339A286398E71D7B651E8D8ECDD\">Link</a> öffnen oder den QR-Code scannen, um unser Repository hinzuzufügen."
```

What about `fdroidrepos://`?

Comment (SailReal) [reply i18n/de.yaml]: Custom scheme aren't nice handled, rechecked it using a few more apps and the result is not good, e.g.:


Selecting or using this URL leads to https://cryptomator.org/android/fdroid/repo?fingerprint=F7C3EC3B0D588D3CB52983E9EB1A7421C93D4339A286398E71D7B651E8D8ECDD in F-Droid which throws an error...

In my opinion, `fdroidrepos://` creates more potential problems for us than the fact that https://static.cryptomator.org/android/fdroid/repo?fingerprint=F7C3EC3B0D588D3CB52983E9EB1A7421C93D4339A286398E71D7B651E8D8ECDD cannot be opened in the browser on the desktop.

**Review by tobihagemann — APPROVED** [review]

**Pull request merged**